specification/hybridcloud/resource-manager/Microsoft.HybridCloud/preview/2022-11-01-preview/hybridcloud.json

```diff
@@ -0,0 +1,992 @@
+          "x-ms-secret": true
+        },
+        "cloudConnections": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ResourceReference"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "List of authorized Resources which have access to this connector"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "#/definitions/ProvisioningState",
+          "description": "The provisioning state of the application rule collection resource."
+        }
+      },
+      "description": "Cloud Connector resource properties."
+    },
+    "CloudConnector": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CloudConnectorPropertiesFormat",
+          "description": "Properties of the Cloud Connector."
+        },
+        "identity": {
+          "allOf": [
+            {
+              "$ref": "../../../../../common-types/resource-management/v4/managedidentity.json#/definitions/ManagedServiceIdentity"
+            }
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Cloud Connector resource."
+    },
+    "ClientSecret": {
+      "type": "object",
+      "description": "Secret to access the AAD Application ID and Customer KeyVault",
+      "properties": {
+        "key": {
+          "type": "string",
+          "description": "Secret to access the AAD Application ID"
+        },
+        "keyVaultId": {
+          "type": "string",
+          "description": "Resource Ref of the Key vault containing the Client Secret / Certificate"
+        }
+      }
+    },
+    "CloudConnectionListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudConnection"
+          },
+          "x-ms-identifiers": [
+            "id"
+          ],
+          "description": "A list of Cloud Connection resources."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to hold the next page of results"
+        }
+      },
+      "description": "Response for the ListCloudConnections API service call"
+    },
+    "CloudConnectionPropertiesFormat": {
+      "type": "object",
+      "properties": {
+        "cloudConnectorId": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The Cloud Connector to which the Cloud Connection belongs."
+        },
+        "virtualHub": {
+          "$ref": "#/definitions/ResourceReference",
+          "description": "The virtualHub to which the Cloud Connection belongs."
+        },
```

Comment:
these could be format arm-id too.
What's their x-ms-mutability? readonlyness?

Reply:
Fixed